langserve/playground/src/App.tsx

```diff
@@ -329,7 +328,7 @@ function App() {
             {latest && (
               <div className="flex flex-col gap-3">
                 <h2 className="text-xl font-semibold">Output</h2>
-                <div className="p-4 border border-divider-700 flex flex-col gap-3 rounded-2xl bg-background text-lg">
+                <div className="p-4 border border-divider-700 flex flex-col gap-3 rounded-2xl bg-background text-lg whitespace-pre-wrap">
```

Comment:
We might want to add `break-words` as well?

Reply:
Done